docs/index.txt

```diff
@@ -9,62 +9,172 @@ Laravel MongoDB
 .. meta::
    :keywords: php framework, odm
 
-This package adds functionalities to the Eloquent model and Query builder for
-MongoDB, using the original Laravel API.
-*This library extends the original Laravel classes, so it uses exactly the
-same methods.*
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+
+   /install
+   /eloquent-models
+   /query-builder
+   /user-authentication
+   /queues
+   /transactions
+   /upgrade
+
+Introduction
+------------
+
+Welcome to the documentation site for the official {+odm-long+}.
```

Comment:
What are the value of `{+odm-long+}` and `{+odm-short+}`?

Reply:
"Laravel MongoDB package" and ""Laravel MongoDB", respectively.
One issue in working with Snooty and submodules is that the config files that define the behavior and source constants (such as "odm-long" and "odm-short" need to be in the docs repo. Here are the current values in a [PR](https://github.com/10gen/docs-laravel/pull/6/files#diff-06c9aca58712658eed71cea4936907e0a123d987fd055627808cb7cc4c5c0e93R10-R13).

When the PR is ready for review, I'll link the info so that this PR can be reviewed more easily.